<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMain_win</class>
 <widget class="QMainWindow" name="MyMain_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1000</width>
      <height>560</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>560</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>560</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="main_page">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>560</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>560</height>
      </size>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>881</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="MainLayout">
       <item>
        <widget class="QLabel" name="Title_label">
         <property name="minimumSize">
          <size>
           <width>435</width>
           <height>141</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>888</width>
           <height>141</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Myanmar Text</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string notr="true">交通咨询系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="MainChoice_Layout">
         <item>
          <widget class="QPushButton" name="PathChoose_pushButton">
           <property name="styleSheet">
            <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:Microsoft Yahei;
    /*字体大小为20点*/
    font-size:20pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(14 , 150 , 254);
    /*边框圆角半径为8像素*/ 
    border-radius:8px;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(44 , 137 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(14 , 135 , 228);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
           </property>
           <property name="text">
            <string>路径选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PathCheck_pushButton">
           <property name="styleSheet">
            <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:Microsoft Yahei;
    /*字体大小为20点*/
    font-size:20pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(14 , 150 , 254);
    /*边框圆角半径为8像素*/ 
    border-radius:8px;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(44 , 137 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(14 , 135 , 228);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
           </property>
           <property name="text">
            <string>车次信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CityCheck_pushButton">
           <property name="styleSheet">
            <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:Microsoft Yahei;
    /*字体大小为20点*/
    font-size:20pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(14 , 150 , 254);
    /*边框圆角半径为8像素*/ 
    border-radius:8px;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(44 , 137 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(14 , 135 , 228);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
           </property>
           <property name="text">
            <string>城市信息</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="PathChoose_page">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>560</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>560</height>
      </size>
     </property>
     <widget class="QPushButton" name="Return_pushButton">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>20</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>837</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="Way_label">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>交通工具</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Way_comboBox">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>29</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Src_label">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>起始城市</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="src_lineEdit">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="des_label">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>终点城市</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="des_lineEdit">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="PreferChoice_label">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>29</height>
              </size>
             </property>
             <property name="text">
              <string>选择</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="PreferChoice_comboBox">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>29</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Check_pushButton">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="start_label">
             <property name="text">
              <string>出发时间</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="month_comboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="month_label">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>月</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="day_comboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="day_label">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>日</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="hour_comboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="month_label_3">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>时</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="min_comboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="month_label_4">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>分</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="PathChoose_Table"/>
       </item>
       <item>
        <widget class="QLabel" name="PathCheckReslabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>450</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PathCheck_page">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>560</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>560</height>
      </size>
     </property>
     <widget class="QPushButton" name="Return_pushButton_2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>60</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="PathCheck_Table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>959</width>
        <height>311</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>959</width>
        <height>311</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>959</width>
        <height>311</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>30</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>车次信息</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>292</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="PathCheck_Add_pushButton">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PathCheck_Del_pushButton">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PathCheck_Mdy_pushButton">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>213</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="PlaneLineSearch_radioButton">
           <property name="text">
            <string>飞机</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="TrainLineSearch_radioButton">
           <property name="text">
            <string>火车</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="Row_label">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>460</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CityCheck_page">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>560</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>560</height>
      </size>
     </property>
     <widget class="QLabel" name="CityCheck_label">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>城市信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Return_pushButton_3">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>50</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>446</width>
        <height>232</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="CityCheck_Table"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="City_Add_pushButton">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="City_Del_pushButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="City_Mdy_pushButton">
           <property name="text">
            <string>修改</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>256</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="CityPlane_radioButton">
         <property name="text">
          <string>飞机</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="CityTrain_radioButton">
         <property name="text">
          <string>火车</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
